src/commands/synthetics/__tests__/reporters/__snapshots__/default.test.ts.snap

```diff
@@ -61,9 +61,9 @@ exports[`Default reporter testEnd 1 API test (non-blocking), 1 location, 3 resul
 `;
 
 exports[`Default reporter testEnd 1 API test, 1 location, 1 result: success 1`] = `
-"[1m[32m✓[39m[22m [[1m[2maaa-aaa-aaa[22m[22m] [1mTest name[22m - [1m[32mlocation: [1mFrankfurt (AWS)[22m[1m[39m[22m
+"[1m[31m✖[39m[22m [[1m[31mblocking[39m[22m] [[1m[2maaa-aaa-aaa[22m[22m] [1mTest name[22m - [1m[31mlocation: [1mFrankfurt (AWS)[22m[1m[39m[22m
   ⎋ Total duration: 123 ms - Result URL: [2m[36mhttps://app.datadoghq.com/synthetics/details/aaa-aaa-aaa?resultId=1&from_ci=true[39m[22m 
-  [1m[32m✓[39m[22m [1m[32m[1mGET[22m[1m - http://fake.url[39m[22m
+  [1m[31m✖[39m[22m [1m[31m[1mGET[22m[1m - http://fake.url[39m[22m
 
 "
 `;
```

Comment:
I think we have some unwanted changes here where some tests are shown as failing because `getApiResult()` sets `passed: false` and it doesn't get updated?

Reply:
Ah good catch, I'll fix that (those snapshots were not easy to read!)

Reply:
Actually I have the feeling it was the test that was broken before 🤔 . Take this one for instance: https://github.com/DataDog/datadog-ci/blob/master/src/commands/synthetics/__tests__/reporters/default.test.ts#L155 it says 3 success but then it passes `passed: false` later on to `createApiPollResult`, so the red cross `x` is legit, and it was actually broken before? @ruizb any idea?

Reply:
It says 3 results, 1 success + 1 failed + 1 failed non-blocking, hence the `passed: false` for some of them 👀 

I think Etienne got it right 🤔

Reply:
It was that 👍 https://github.com/DataDog/datadog-ci/pull/543/commits/a30d61c2aa0012c179637342660455b4261b77df#diff-f8505416f94fe5414df0ab9e792e8fd3a8ebd956950ac9574ce67b9a52ac9cdaR142